### PR TITLE
docs: kb on transient language code identifiers on Keyman for Windows

### PR DESCRIPTION
**Pull request by Meng-Heng**
This PR is ready for review.

### CODE REVIEW

File: knowledge-base/kb0116.md [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+
+1. Open Command Prompt as administrator.
+
+2. Run the command (the final = sign is important):
+
+    `"C:\program files (x86)\keyman\keyman desktop\kmshell.exe" -i armenian.kmp=` 
+
+    (This will install armenian.kmp for all users, but with no language association.)
+    
+    *If an error occurs, please refer to this [article](/kb/117).*
```
Comment (Meng-Heng) [inline knowledge-base/kb0116.md:78]: Can I write another kb on this?

Comment (darcywong00) [reply knowledge-base/kb0116.md:78]: Maybe kb117 can go on separate PR, and move this link to that new PR

File: knowledge-base/kb0116.md [hunk 2]
```diff
@@ -0,0 +1,96 @@
+# HOWTO: Working with transient languages on Keyman for Windows
+
+This article will help solve the error for the limitation of 4 custom language tags on Keyman.
+![Add a language to Keyboard's error](./assets/kb0116/add_language_to_keyboard_error.png)
```
Comment (mcdurdin) [inline knowledge-base/kb0116.md:4]: We can't really solve the error -- it's a limitation that we need to explain. 

```suggestion
This article will explain the Windows limitation of 4 custom language tags that may result in the display of a message when you attempt to install a Keyman keyboard: "Unable to install keyboard language; Windows has a limit of 4 custom 'transient' languages, and you may have reached this limit."
![Add a language to Keyboard's error](./assets/kb0116/add_language_to_keyboard_error.png)
```

File: knowledge-base/kb0116.md [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+### Transient LCIDs
+
+If there are four identifiers, those are likely specified as shown:
+
+```c
+0x2000: und-Latn (Undetermined, Latin script) – often used for IPA
+0x2400: tk-Arab-AF (Turkmen, Arabic script, Afghanistan) – Turkmen in Latin script is available as a non-transient language, but not Arabic script
+0x2800: wbl-Arab-Af (Wakhi, Arabic script, Afghanistan)
+0x2C00: grc-Grek-GR (Ancient Greek, Greek script, Greece)
+```
```
Comment (mcdurdin) [inline knowledge-base/kb0116.md:61]: There are many possible transient languages -- the usage will vary dramatically; we should remove this section.

File: knowledge-base/kb0116.md [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+
+More on transient LCIDs in the [Published Version](https://learn.microsoft.com/en-us/openspecs/windows_protocols/ms-lcid/70feba9f-294e-491e-b6eb-56532684c37f#published-version) section.
```
Comment (mcdurdin) [inline knowledge-base/kb0116.md:63]:
```suggestion
More information on LCIDs is available in the [Microsoft LCID documentation](https://learn.microsoft.com/en-us/openspecs/windows_protocols/ms-lcid/70feba9f-294e-491e-b6eb-56532684c37f#published-version) section.
```

File: knowledge-base/kb0116.md [hunk 5]
```diff
@@ -0,0 +1,96 @@
...
+
+## Workaround
+You may need to install the keyboards without a language association (or associated with EN) and allow the users to setup which languages they wish to use.
```
Comment (mcdurdin) [inline knowledge-base/kb0116.md:66]:
```suggestion
You may need to install the keyboards without a language association (or associated with EN), or remove languages that you are no longer using. 
```

File: knowledge-base/kb0116.md [hunk 6]
```diff
@@ -0,0 +1,86 @@
...
+```
+
+More information on LCIDs is available in the [Microsoft LCID documentation](https://learn.microsoft.com/en-us/openspecs/windows_protocols/ms-lcid/70feba9f-294e-491e-b6eb-56532684c37f#published-version) section.
+
+## Workaround
+You may need to install the keyboards without a language association (or associated with EN), or remove languages that you are no longer using. 
+
+### For example: With armenian.kmp
+
+1. Open Command Prompt as administrator.
```
Comment (rc-swag) [inline knowledge-base/kb0116.md:60]: maybe link to kb0117

File: knowledge-base/kb0116.md [hunk 7]
```diff
@@ -0,0 +1,86 @@
...
+    CachedLanguageName    REG_SZ    @Winlangdb.dll,-1650
+    0804:{81D4E9C9-1D3B-41BC-9E6C-4B40BF79E35E}{FA550B04-5AD7-411F-A5AC-CA038EC515D7}    REG_DWORD    0x1
+```
+
+More information on LCIDs is available in the [Microsoft LCID documentation](https://learn.microsoft.com/en-us/openspecs/windows_protocols/ms-lcid/70feba9f-294e-491e-b6eb-56532684c37f#published-version) section.
+
+## Workaround
+You may need to install the keyboards without a language association (or associated with EN), or remove languages that you are no longer using. 
+
+### For example: With armenian.kmp
```
Comment (rc-swag) [inline knowledge-base/kb0116.md:58]:
```suggestion
### Associated with different language example: With armenian.kmp
```

File: knowledge-base/kb0116.md (hunk 6 above)
Comment (Meng-Heng) [reply knowledge-base/kb0116.md:60]: Yes, should I link it before the kb is available?

**Review by rc-swag — APPROVED** [review]
lgtm

**Pull request merged**